archlinux/README-short.txt

```diff
@@ -0,0 +1 @@
+Archlinux is a simple, lightweight Linux distribution aimed for flexibility.
```

Comment:
Was "Archlinux" here intentional? (I would've expected either "archlinux" or "Arch Linux", but I'll admit I'm not familiar with what's canonical within the Arch community here :sweat_smile:)

Reply:
You're right. I quickly sketched this out and muscle memory has made me make mistakes more often than not ;P

Reply:
I think you missed this one. :sweat_smile: